installers/ansible/roles/pgo-operator/files/crds/perconapgclusters-crd.yaml

```diff
@@ -26,8 +26,12 @@ spec:
             pgbadgerport: { type: string }
             PrimaryStorage: { type: object }
             port: { type: string }
```

Comment:
let's remove properties section. 
Since it is irrelevant to our object and will be reflected on Openshift Web page as is.